x/psiphon/go.mod

```diff
@@ -0,0 +1,21 @@
+module github.com/Jigsaw-Code/outline-sdk/x/psiphon
+
+go 1.20
+
+// Use our non-functional stub implementation, instead of the official GPL one.
+// Actual users will have to depend on the official GPL code instead.
+replace github.com/Psiphon-Labs/psiphon-tunnel-core => ./stub
```

Comment:
Hmm, this is ugly. I'm wondering if we put both `x/psiphon` and `x/examples/fetch-psiphon` to another repository (like `outline-sdk-examples`), can we mitigate the GPL license issue?

Reply:
I'm actually thinking we can avoid all of this, but I need to consult with the licensing team. I think GPL only applies when you distribute software, and we are not doing that. It would apply if we start distributing the mobileproxy as a library though. Needs investigation.

I kept the adaptor separate from the example app on purpose. The adaptor is standalone. We  can write the code and even test. The fetch app only makes sense with the GPL code pulled into it. If the fetch app was to be distributed, it would have to be GPL, but not the adaptor.